pkg/api/api.go

```diff
@@ -197,13 +199,19 @@ func (a *API) RegisterOverridesExporter(oe *exporter.OverridesExporter) {
 // RegisterDistributor registers the endpoints associated with the distributor.
 func (a *API) RegisterDistributor(d *distributor.Distributor) {
 	pyroscopeHandler := pyroscope.NewPyroscopeIngestHandler(d, a.logger)
+	otlpHandler := otlp.NewOTLPIngestHandler(d, a.logger)
 	a.RegisterRoute("/ingest", pyroscopeHandler, true, true, "POST")
 	a.RegisterRoute("/pyroscope/ingest", pyroscopeHandler, true, true, "POST")
 	pushv1connect.RegisterPusherServiceHandler(a.server.HTTP, d, a.grpcAuthMiddleware)
 	a.RegisterRoute("/distributor/ring", d, false, true, "GET", "POST")
 	a.indexPage.AddLinks(defaultWeight, "Distributor", []IndexPageLink{
 		{Desc: "Ring status", Path: "/distributor/ring"},
 	})
+
+	pprofileotlp.RegisterGRPCServer(a.server.GRPC, otlpHandler)
```

Comment:
Just so you know I don't think we expose GRPC endpoint anywhere in our infra nor in helm.

Reply:
@cyriltovena yeah we might need a separate grpc server